README.md

```diff
@@ -1,94 +1,78 @@
 # phonegap-plugin-multiview
+***
 
-This plugin allows your PhoneGap app to create multiple views.
+This plugin allows your PhoneGap app to create multiple views.  This is only Version 1 of the MultiView Plugin, stay tuned because we will be continually adding more functionality as we incorporate feedback!
```

Comment:
This is 'only' makes it sounds like this is not ready. An active project should not need to market itself as having more coming.